packages/tuono-router/src/components/RouteMatch.spec.tsx

```diff
@@ -64,11 +75,38 @@ describe('<RouteMatch />', () => {
           <div
             data-testid="current"
           >
-            current route
+            {"some":"data"}
           </div>
         </div>
       </div>
     `,
     )
   })
+
+  it('should return null data when while loading', () => {
+    vi.mocked(useServerPayloadData).mockReturnValue({
+      data: { some: 'data' },
+      isLoading: true,
+    })
```

Comment:
the test title says: "should return null data when while loading"
however the mock is returning `data` set to `{ some: 'data' }` which isn't the correct value
```suggestion
    vi.mocked(useServerPayloadData).mockReturnValue({
      data: null,
      isLoading: true,
    })
```

Reply:
The goal of the test is to return `null` even when `data` exists (due to a routing transition)